makedist

```diff
@@ -45,19 +45,24 @@ usage()
     echo "   directory as the script."
     echo " The -s option makes both operations work for singcvmfs instead"
     echo "   of cvmfsexec, and files are left in .singcvmfs"
+    echo " The -v option allows you to select a CVMFS version. In the format X.Y.Z"
```

Comment:
Also add `[-v version]` to the `Usage` line (and please remove the blanks inside the brackets of the `-m` option too).